galaxy/Dockerfile

```diff
@@ -125,6 +125,7 @@ RUN ansible-playbook /ansible/postgresql_provision.yml && \
     --extra-vars galaxy_extras_config_condor=True \
     --extra-vars galaxy_extras_config_condor_docker=True \
     --extra-vars galaxy_extras_config_rabbitmq=True \
+    --extra-vars galaxy_extras_config_cvmfs=True \
```

Comment:
This is why the tool data table is not configured for the single image - it's missing the config definition here:
`--extra-vars galaxy_tool_data_table_config_file=/etc/galaxy/tool_data_table_conf.xml \`